index.js

```diff
@@ -16,15 +13,26 @@ async function main() {
         await exec.exec('pip', ['freeze', '--local']);
     });
 
+    let args = core.getInput('extra_args');
+    if (!args) {
```

Comment:
since there's a default we can remove the `!args` case and always splat into the array (please splat at the end since it makes it the most consistent / compatible for invocations using `--files`)

Reply:
@asottile Done! Added a couple alternatives [here](https://github.com/pre-commit/action/pull/15#discussion_r425553602), as I wasn't sure if prefer any of them stylistically/if they're more consistent with the other assignments.

Reply:
either would be better than the current code which changes the type of `args` from String to Array

Reply:
Second option applied.